README.md

```diff
@@ -14,19 +14,29 @@
 - Use a function to iterate over tasks
 - Add basic styling
 
+### Milestone 2:
+
+- Add drag and drop javascript files
+- Add methods for status updates
+- Import new files as modules
+- Add event listeners to list items
+- Make object values update based on index and completed tasks
+- Add strikethrough styling
+- Store items in local storage
+
 ## Built With
 
 - HTML, SCSS, Bootstrap, Webpack, Javascript
 
 ## Live Demo
 
-[Live Demo Link](https://ghiftee.github.io/To-do-list/dist/)
+[Live Demo Link](https://raw.githack.com/Ghiftee/To-do-list/Interactive-list/dist/index.html)
 
 ## Getting Started
 
 To get a local copy up and running:
 
-You should have basic knowledge of HTML,Webpack, Javascript, a text editor and chrome browser.
+You should have basic knowledge of HTML, Webpack, Javascript, a text editor and chrome browser.
```

Comment:
[OPTIONAL]

- I suggest you to add the command as well on how to clone it, and also how to run the project in local. Think about the user who doesn't have technical knowledge.